src/validation/mo_sethet/CMakeLists.txt

```diff
@@ -0,0 +1,64 @@
+set(SETHET_VALIDATION_DIR ${MAM_X_VALIDATION_DIR}/mo_sethet)
+set(SETHET_VALIDATION_SCRIPTS_DIR ${MAM_X_VALIDATION_DIR}/scripts)
+
+
+# These subdirectories contain Skywalker drivers for MAM4 parameterizations.
+# Include directory for .mod files.
+
+include_directories(${PROJECT_BINARY_DIR}/validation)
+
+# We use a single driver for all sethet-related parameterizations.
+add_executable(sethet_driver
+               sethet_driver.cpp
+               calc_het_rates.cpp
+               calc_precip_rescale.cpp
+               find_ktop.cpp
+               gas_washout.cpp
+               )
+
+target_link_libraries(sethet_driver skywalker;validation;${HAERO_LIBRARIES})
+
+# Copy some Python scripts from mam_x_validation to our binary directory.
+foreach(script
+        compare_mam4xx_mam4.py)
+  configure_file(
+    ${SETHET_VALIDATION_SCRIPTS_DIR}/${script}
+    ${CMAKE_CURRENT_BINARY_DIR}/${script}
+    COPYONLY
+  )
+endforeach()
+# stand_calc_sum_wght_ts_355
+# Run the driver in several configurations to produce datasets.
+set(TEST_LIST
+    calc_het_rates_ts_355
+    calc_precip_rescale_ts_355
+    find_ktop_ts_355
+    gas_washout_ts_355
+    )
+# # matching the tests and errors, just for convenience
+
+set(DEFAULT_TOL 1e-13)
+set(ERROR_THRESHOLDS
+    ${DEFAULT_TOL}
+    ${DEFAULT_TOL}
+    ${DEFAULT_TOL} # output is an int
+    9e-3
```

Comment:
This tolerance is quite a bit higher than the default. Is there something fancy going on in the `gas_washout` logic? Are you confident that we're not missing something there?

Reply:
The answers get increasingly larger - up to around 1e10 - the answers look visually close to the extent that they are printed so I think the logic is in line with the fortran